catalog-info.yaml

```
@@ -0,0 +1,11 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: habitat
+  description: Casavo Design System
```

```suggestion
metadata:
  name: habitat
  description: Casavo Design System
  annotations:
    github.com/project-slug: casavo/habitat
```

this extra annotation adds some magic linking :)